tests/integration/docker_utils/test_docker.py

```diff
@@ -122,6 +122,9 @@ def test_container_lifecycle_commands(self, docker_client: ContainerClient):
             docker_client.unpause_container(container_id)
             assert DockerContainerStatus.UP == docker_client.get_container_status(container_name)
 
+            docker_client.restart_container(container_id)
```

Comment:
I guess it would be nice to add _some_ test to check if the container actually restarted. An easy example would be printing something (like date) and then going into a sleep loop, then restarting, and checking if the log out put is there twice.